cypress/cypress-integration/integration/pending-report.cy.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  FORMATS,
+  formatNow,
+} from '../../../shared/src/business/utilities/DateHandler';
+import { loginAsDocketClerk } from '../../helpers/auth/login-as-helpers';
+import path from 'path';
+
+describe('Pending report', () => {
+  const today = formatNow(FORMATS.MMDDYYYY_UNDERSCORED);
+  const fileName = `Pending Report - Chief Judge ${today}.csv`;
+  const downloadsFolder = Cypress.config('downloadsFolder');
+
+  describe('View pending report', () => {
+    it('should log in as a docket clerk and navigate to the pending report then select chief judge from the dropdown', () => {
+      loginAsDocketClerk();
+      cy.get('[data-testid="dropdown-select-report"]').click();
+      cy.get('[data-testid="select-pending-report"]').click();
+      cy.get('[data-testid="dropdown-select-judge"]').select('Chief Judge');
+    });
+
+    it('should have more rows in the table after clicking the "load more" button than before', () => {
```

Comment:
Hmm, this assumes data will be prepopulated in the system. Will this work for a newly created environment that has no data in the pending report?

Reply:
We have 113 pending items in our seed data, which should mean that this will always work for an integration test. Is that correct?

Reply:
hm, I guess I mistakenly thought this would be run upon a deployed environment... and if we wiped an environment clean, there wouldn't be any items in a pending report. Upon closer look, and thanks to your comment, this is only testing locally upon our seed data.